version.go

```diff
@@ -1,4 +1,4 @@
-package version
+package interchaintest
```

Comment:
Should we leave this as an internal package also? I don't see why someone would need access to this outside of `interchaintest`

Reply:
reverted: https://github.com/strangelove-ventures/interchaintest/pull/1102/commits/7be6bd7100724cc8f11f901f1d7789236e5c6885